Homework7/Wizard.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Homework7
+{
+    public class Wizard : Unit
+    {
+        private const double HP = 150;
+        private const double AD = 50;
+        private const double ATTACK_DAMAGE_PERCENT = 0.32;
+
+        private const double DODGE_ATTACK_CHANCE = 0.5;
+        private const double UNSUCCESS_DODGE_CHANCE = 1.1;
+
+        public Wizard() : base(AD, HP)
+        {
+        }
+
+        protected override double Attack()
+        {
+            double attackDamage = 0;
+
+            if (Randomizer.IsSuccessAction(ATTACK_DAMAGE_PERCENT))
```

Comment:
Are you sure about this logic?
Am I right that he tries to attack and if he missed, he tried again?

Reply:
yes, it's wizzard's capabilities